index_websocket.js

```diff
@@ -0,0 +1,97 @@
+const http = require("http");
+const open = require("open");
+const fs = require("fs");
+const WebSocketServer = require("websocket").server; // websocket server class
+
+let webSocket = null;
+// connection variable, to refer to the web socket connection to the client once connection is established
+let connection = null;
+// default port used if none is specified by the user
+let defaultPort = 9101;
+const host = "http://127.0.0.1";
+
+// check if a browser window is already open
+let isRunning = false;
+
+
+exports.visualise = function visualise(jsonData, port = defaultPort) {
```

Comment:
Let's add a comment block for all functions. 
Example:
<img width="565" alt="image" src="https://user-images.githubusercontent.com/19206398/83043495-8c149b80-a075-11ea-873b-1f795c9c5d0a.png">